test-js/unit/pentaho/visual/base/View.Spec.js

```diff
@@ -0,0 +1,57 @@
+define([
+  "pentaho/visual/base/View"
+], function(View) {
+  "use strict";
+
+  describe("pentaho/visual/base/View", function() {
+    var element, dataSpec;
+
+    beforeEach(function() {
+      element = document.createElement("div");
+      var data = {};
+      dataSpec = {v: data};
+    });
+
+    describe("the constructor ", function() {
+
+      it("should throw if not invoked with two arguments", function() {
+        expect(function() {
+          return new View();
+        }).toThrowError(/required/);
+
+        expect(function() {
+          return new View(element);
+        }).toThrowError(/required/);
+
+        expect(function() {
+          return new View(element, dataSpec);
```

Comment:
Sending in an arbitrary object into the `model` argument doesn't seem like good idea. It is true that you and I know that the current code does not validate the type of thing received, or uses it in the constructor somehow, but relying on that on a unit test looks like a brittle assumption and unnecessary risk.